[terra-tabs]  Fix for the focus traverse within the tab content. (#1778)
cerner/terra-framework: packages/terra-tabs/CHANGELOG.md

## Unreleased

> * Changed
>   * Fix for the focus traverse within the tab content .

Reviewer: ```suggestion
* Added
  * Added prop to support the focus traverse within the tab content.
```
